### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by edenhill**

### CODE REVIEW

**Comment by edenhill** [comment]
@mhowlett Can I please get a review on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,19 @@ and the sticky consumer group partition assignor.
...
+   endless loop or generate uneven assignments in case the group members
+   had asymmetric subscriptions (e.g., c1 subscribes to t1,t2 while c2
+   subscribes to t2,t3).  (#3159)
+
+
+### Producer fixes
+
+ * `linger.ms` (if not configured) could be auto-adjusted on `topic_new()`
+   if `message.timeout.ms` was set in the topic-specific configuration.
+   This is now only done based on the default_topic_conf at producer creation.
```
Comment (mhowlett) [inline CHANGELOG.md:65]: FWIW, i'm confused after reading this text.

**Review by mhowlett — APPROVED** [review]
can't see anything bad

File: CHANGELOG.md (hunk 1 above)
Comment (edenhill) [reply CHANGELOG.md:65]: Fixed

**Pull request merged**